velox/exec/tests/DriverTest.cpp

```diff
@@ -932,6 +932,18 @@ class ThrowOperator : public Operator {
   const ThrowNode::OperatorMethod throwingMethod_;
 };
 
+class ThrowIsBlockedOperator : public ThrowOperator {
+ public:
+  ThrowIsBlockedOperator(
+      DriverCtx* ctx,
+      int32_t id,
+      const std::shared_ptr<const ThrowNode>& node)
+      : ThrowOperator(ctx, id, node) {}
+  BlockingReason isBlocked(ContinueFuture* /*future*/) override {
+    return BlockingReason::kYield;
```

Comment:
perhaps, add a comment to explain what's going on.

// Report being blocked, but do not set the future.